Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -180,8 +183,8 @@ def remove_tid(dataset):
     """ Remove TaskID (_tidXX) part from dataset name.
 
     As AMI GetPhysicsParamsForDataset works with containers,
-    we construct the container name from each dataset name,
-    removing the _tid{...} part
+    the container name is extracted from each dataset name by
+    removing the _tid{...} part.
```

Comment:
Well, talking about this function... Information about GetPhysicsParamsForDataset does not seem to be a relevant description at all, however reworded. Maybe it deserves renaming?
Something like this:
```
def container_name(dataset):
    """ Construct contained name from dataset name.

    The container name is extracted from each dataset name by
    removing the _tid{...} part.

    :param ...: ...
    ...
    """
```

Reply:
And, I am not sure about curly braces here: `...removing the _tid{...} part.`

I mean, why curly? In regular expressions they are used to set number of repetitions; it may also refer to bash [brace expansion](https://www.linuxjournal.com/content/bash-brace-expansion)... and only after realizing it is none of the above I would start thinking about some custom meaning.
Maybe replace them with asterisk? It looks like more or less standard wildcard symbol: `...removing '_tid*' part`. Or, simply leave omission points: `...removing '_tid...' part`.

Not a big deal, of course, but since you raise docs improvement question for this stage...;)

Reply:
Agreed on both points. Done.